experimental/dds/tree2/src/feature-libraries/editable-tree/editableTreeTypes.ts

```diff
@@ -328,6 +328,16 @@ export interface EditableField
 	 */
 	insertNodes(index: number, newContent: ITreeCursor | ITreeCursor[]): void;
 
+	/**
+	 * Moves nodes from this field to destination.
```

Comment:
This needs to clarify that its only supported between sequence fields, and what it does when the destinationField  is not specified.

Reply:
updated doc comment